backend/src/services/worker/worker.service.ts

```diff
@@ -160,6 +160,16 @@ class Worker {
     const workers = await this.#workerRepository.getAll(param);
     return { items: workers };
   }
+
+  async getWorkersIdsByTenant(tenantId: string): Promise<string[]> {
+    const workers = await this.#workerRepository.getWorkersByTenant(tenantId);
+
+    const workersIds = workers.map((worker) => {
+      return worker.id;
+    });
+
+    return workersIds;
+  }
```

Comment:
Do you really need this method? It is not an edpoint handler. It also does not contain any logic. I'd better use `this.#workerRepository.getWorkersByTenant` anywhere you want to get workers for tenant